### DISCUSSION TITLE
Welcome to iot-communication Discussions!

### DISCUSSION THREAD

**Discussion by xingshuangs** [xingshuangs/iot-communication#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brunom1265** [comment]
It's possible to connect to Siemens PLC with password? That we cant put a password to the plc, because some plc are protect, and you cant read or write

**Reply by brunom1265** [reply]
And was nice to read analog inputs and outputs

**Reply by xingshuangs** [reply]
Yes, you can connect to Siemens PLC without password.

**Reply by brunom1265** [reply]
> Yes, you can connect to Siemens PLC without password.

I was asking if is posible to connect with a plc that is protected with password

**Reply by xingshuangs** [reply]
I'm not sure. You can try it yourself
